Intersect.Server/Entities/Player.cs

```diff
@@ -1401,50 +1417,76 @@ public bool TryGiveItem(Item item, bool bankOverflow, bool sendUpdate)
                 return false;
             }
 
+            // is this item stackable?
             if (itemBase.IsStackable)
```

Comment:
As part of the effort to stop calling everything "base" can you rename the variable to `itemDescriptor`?